packages/zero-cache/src/custom-queries/transform-query.ts

```diff
@@ -164,10 +127,3 @@ export class CustomQueryTransformer {
     }
   }
 }
-
-function getCacheKey(headerOptions: HeaderOptions, queryID: string) {
-  // For custom queries, queryID is a hash of the name + args.
-  // the APIKey from headerOptions is static. Not needed for the cache key.
-  // The token is used to identify the user and should be included in the cache key.
-  return `${headerOptions.token}:${headerOptions.cookie}:${queryID}`;
```

Comment:
looks fine to keep the cache given we encode these options. The only edge case left would be code updates but 5 seconds is a small amount of time.